ocs/ocp.py

```diff
@@ -152,6 +163,21 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
```

Comment:
Did you miss to delete this line?

Reply:
no, why?
This is the command for apply

Reply:
if my understanding is correct, this nullifies whatever we set for  "command" in the previous lines.

Reply:
the misunderstanding was due to the fact that there were hidden lines of code :-) Thanks @RazTamir for pointing out that! LGTM.